From: Paolo Valente <paolo.valente@linaro.org>
Date: Mon, 3 Feb 2020 11:41:00 +0100
Subject: [PATCH] block, bfq: clarify the goal of bfq_split_bfqq()
References: bsc#1175995,jsc#SLE-15608
Git-commit: c92bddee77353a773f0df76115c1a01877ce1eae
Patch-mainline: v5.6-rc1

The exact, general goal of the function bfq_split_bfqq() is not that
apparent. Add a comment to make it clear.

Tested-by: Oleksandr Natalenko <oleksandr@natalenko.name>
Signed-off-by: Paolo Valente <paolo.valente@linaro.org>
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 block/bfq-iosched.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/block/bfq-iosched.c b/block/bfq-iosched.c
index 50b73d9893ec..e74fd7f7e4b9 100644
--- a/block/bfq-iosched.c
+++ b/block/bfq-iosched.c
@@ -5992,6 +5992,8 @@ static void bfq_finish_requeue_request(struct request *rq)
 }
 
 /*
+ * Removes the association between the current task and bfqq, assuming
+ * that bic points to the bfq iocontext of the task.
  * Returns NULL if a new bfqq should be allocated, or the old bfqq if this
  * was the last process referring to that bfqq.
  */
-- 
2.16.4

